NoQuD/tests/AI_3plus_test.csv

```diff
@@ -0,0 +1,30 @@
+Number of Assembly Types,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
Is this test for the assembly input, is that what AI means? Perhaps a clearer name for your test would help.

Reply:
Yeah, it stands for assembly information. I'll see about changing the input file name.